scripts/run_chain_tests.py

```diff
@@ -26,8 +31,11 @@ def run_chain_test(chain):
         return -1
 
     os.environ["SAS_SUBSTRATE_WS_URL"] = url
-    proc = subprocess.Popen(["yarn", "dev"], stdout=subprocess.PIPE,
-                            stderr=subprocess.STDOUT)
+    proc = subprocess.Popen(
+        ["yarn", "dev"],
```

Comment:
```suggestion
        ["yarn", "build", "&&", "yarn", "start"],
```

Just realized this but I think it will also to help to have this not in dev mode so were not using tsc-watch, but instead just running the build itself.

Reply:
I don't think you can use `&&` in that case as it's a shell thing; we'd have to run them separately or use sh/bash like `["sh", "-c", "yarn build && yarn start"]`

Reply:
I'll merge this as-is for now, but happy to make a change like the above (or just spawning both processes, one after the other) in a little separate PR if you like :)